draft-ietf-tls-tls13.md

```diff
@@ -2204,7 +2186,9 @@ signature
   The values indicate anonymous signatures, RSASSA-PKCS1-v1_5
   {{RFC3447}}, DSA {{DSS}}, and ECDSA {{ECDSA}}, respectively.  The
   "anonymous" value is meaningless in this context but used in
-  {{server-key-share}}.  It MUST NOT appear in this extension.
+  {{server-key-share}}. It MUST NOT appear in this extension.
```

Comment:
minor tweak suggestion: merge these two sentences to drop the "it" for slightly better clarity, e.g.
> The
>  "anonymous" value is meaningless in this context, but used in
>  {{server-key-share}}, and MUST NOT appear in this extension.